addons/stock_account/models/account_move.py

```diff
@@ -233,7 +233,7 @@ def _stock_account_anglo_saxon_reconcile_valuation(self, product=False):
                     # Search for anglo-saxon lines linked to the product in the stock moves.
                     product_stock_moves = stock_moves.filtered(lambda stock_move: stock_move.product_id == prod)
                     product_account_moves |= product_stock_moves._get_all_related_aml().filtered(
-                        lambda line: line.account_id == product_interim_account and not line.reconciled
+                        lambda line: line.account_id == product_interim_account and not line.reconciled and not line.move_id.state == "draft"
```

Comment:
I would rather do this
```suggestion
                        lambda line: line.account_id == product_interim_account and not line.reconciled and line.move_id.state == "posted"
```
As it's cleaner when reading and makes more sense (we just want the posted ones, the cancelled ones are useless too)